### PR TITLE
feature: components for pure react articles

### PR DESCRIPTION
**Pull request by egreb**
#### Please tick a box ###
- [x] **feature** _(A new feature_)
- [ ] **fix** _(A bug fix_)
- [ ] **refactor** _(A code change that neither fixes a bug nor adds a feature_)
- [ ] **docs** _(Documentation only changes_)
- [ ] **performance** _(A code change that improves performance_)
- [ ] **style** _(Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)_)
- [ ] **build** _(Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)_)
- [ ] **ci** _(Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)_)
- [ ] **test** _(Adding missing tests or correcting existing tests_)

#### If you're adding a feature, is it documented in a storybook story?
- [x] Yes
- [ ] No
- [ ] Not adding a new feature

#### Are the components you're working on reusable between brands?
- [ ] Yes _(Using variables that are defined in the default theme)_
- [x] No _(I hard coded some values)_
- [ ] Not working on any components

#### If you're creating markup, did you add proper semantics? 
- [ ] I added [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)
- [ ] I added microdata from [Schema.org](http://schema.org/)
- [x] Semantics are derived from HTML
- [ ] Not applicable

_(Did you do a CR and see that there is something that we should check for each PR, that are not on the list, please [update this document](https://github.com/dbmedialab/shiny/edit/master/.github/PULL_REQUEST_TEMPLATE.md))_

### CODE REVIEW

File: src/atoms/Paragraph.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'react-emotion';
+import propTypes from 'prop-types';
+
+const P = styled.p`
+	font-size: 1.8rem;
```
Comment (misund) [inline src/atoms/Paragraph.js]: Did you consider adjusting BodyText instead of introducing this component?

File: src/atoms/Article.js [hunk 2]
```diff
@@ -1,8 +1,13 @@
-import styled from 'react-emotion';
+import styled, { css } from 'react-emotion';
 import { getVariable } from '../utils';
 
-const Article = styled.article`
+const articleCss = props => css`
```
Comment (misund) [inline src/atoms/Article.js]: I don't understand why you need to export `articleCss`, this looks like an antipattern.

File: src/atoms/Paragraph.js (hunk 1 above)
Comment (misund) [reply src/atoms/Paragraph.js]: Also, this component should not hardcode font-size, font-weight, margin-bottom, but use theme variables  instead: uiRegularSize, uiWeight, uiWeightBold, verticalBase.

File: src/molecules/LazyProgressiveImage.js [hunk 3]
```diff
@@ -55,10 +80,16 @@ class LazyProgressiveImage extends PureComponent {
 			ratio,
 			fadeIn,
 			children,
+			figCaption,
 			preventBlur,
 			backgroundColor,
 		} = this.props;
 
+		let figCaptionElement = null;
+		if (figCaption.intro && figCaption.text) {
```
Comment (misund) [inline src/molecules/LazyProgressiveImage.js]: prefer const

```js
const figcaptionElement = (!figcaption.intro || !figCaptionText ) ? null : (...);
```

File: src/atoms/Article.js (hunk 2 above)
Comment (misund) [reply src/atoms/Article.js]: Still unnecessary changes in this file. Consider rebasing onto master?

Comment (egreb) [reply src/atoms/Article.js]: These changes are needed to make the floating effect on plugs in articles work.

**Pull request merged**